.github/workflows/dart_ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         if: always() && steps.install.outcome == 'success'
 
       - name: Analyze project source
-        run: dart run dart_dev analyze
+        run: dart analyze
```

Comment:
switched to dart analyze since dart_dev still runs dartanalyzer and that is gone in newer Dart
```
[SEVERE] Uncaught Exception:
ProcessException: No such file or directory
  Command: dartanalyzer .
#0      _ProcessImpl._start (dart:io-patch/process_patch.dart:401:33)
#1      Process.start (dart:io-patch/process_patch.dart:38:20)
#2      runProcessAndEnsureExit (package:dart_dev/src/utils/run_process_and_ensure_exit.dart:10:33)
#3      AnalyzeTool.run (package:dart_dev/src/tools/analyze_tool.dart:77:12)
#4      DevToolCommand.run (package:dart_dev/src/dart_dev_tool.dart:154:38)
#5      CommandRunner.runCommand (package:args/command_runner.dart:209:27)
#6      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:119:25)
#7      new Future.sync (dart:async/future.dart:302:31)
#8      CommandRunner.run (package:args/command_runner.dart:119:14)
#9      DartDevRunner.run (package:dart_dev/src/dart_dev_runner.dart:46:35)
#10     runWithConfig (package:dart_dev/src/executable.dart:177:44)
#11     main (file:///Users/robbecker/code/w_flux/.dart_tool/dart_dev/run.dart:9:20)
#12     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:32)
#13     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
```